test/unit/traversing.js

```diff
@@ -743,6 +743,49 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
+	assert.expect( 6 );
+
+    jQuery( "#qunit-fixture" ).append(
```

Comment:
Sounds easier to create node, assign, append and use through the tests?

Like so –
```js
var template = jQuery("<template id='template'>...")
```

Note the single quotes inside the string